app/assets/stylesheets/shared/tooltips.scss

```diff
@@ -0,0 +1,76 @@
+.tooltip-item {  
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;  
+  $tooltip-background: white;
+  $tooltip-color: $base-font-color;
+  $tooltip-max-width: 16em;
+  $tooltip-arrow-width: 8px;
+  $tooltip-shadow: 0 2px 2px silver;
```

Comment:
Color `silver` should be written in hexadecimal form as `#c0c0c0`<br>Color literals like `silver` should only be used in variable declarations; they should be referred to via variable everywhere else.